vizro-ai/docs/pages/user-guides/run-vizro-ai.md

```diff
@@ -81,36 +81,97 @@ There are two ways to integrate Vizro-AI into an application, directly and by ac
             ```
 
 
-2. Vizro-AI's `_get_chart_code` method returns a string of Python code that manipulates the data and creates the visualization. Vizro-AI validates the code to ensure that it is executable and can be integrated.
+2. When the `return_elements` argument of VizroAI's `plot` method is set to `True`, the method returns a `PlotOutputs` data class which contains all possible `VizroAI.plot()` outputs.
+By setting `return_elements=True` you can access code or the figure object. Vizro-AI validates the code to ensure that it is executable and can be integrated.
 
-    !!! example "Application integration via chart code"
+    !!! example "Accessing outputs from PlotOutputs data class"
 
         === "app.py"
-            ```py
-            import vizro.plotly.express as px
-            from vizro_ai import VizroAI
+           ```py
+           import vizro_ai
+           from vizro_ai import VizroAI
+           import vizro.plotly.express as px
+           from dotenv import load_dotenv
 
-            vizro_ai = VizroAI()
+           load_dotenv()
 
-            df = px.data.gapminder()
-            code_string = vizro_ai._get_chart_code(df, "describe life expectancy per continent over time")
-            ```
-        === "code_string"
-            [![ResultCode]][ResultCode]
+           df = px.data.gapminder()
+           vizro_ai = VizroAI()
+
+           plot_outputs = vizro_ai.plot(df, "describe life expectancy per continent over time", explain=True, return_elements=True)
+           fig = plot_outputs.figure
+           code_string = plot_outputs.code
+           ```
+
+Alternatively, you can access code by Vizro-AI's `_get_chart_code` method which returns a string of Python code that manipulates the data and creates the visualization.
```

Comment:
```suggestion
You can also access code with Vizro-AI's `_get_chart_code` method, which returns a string of Python code that manipulates the data and creates the visualization.
```

Reply:
I would have expected Vale to flag the word "alternatively" to warn you it's more complex than using "also" but it obviously hasn't. There are a couple of other places I'd have thought it would warn you (the word encapsulate for example) but as it hasn't, i'll wait until this merges and spend some time in the next few days looking to see why it's not running as expected. Don't want to hold you up and this text is fine as is anyway, even if some usage would be considered more complex than typical.